src/apps/startups/models.py

```diff
@@ -0,0 +1,47 @@
+import uuid
+
+from django.core.validators import RegexValidator
+from django.db import models
+from django.utils import timezone
+
+from apps.users.models import User, Founder
+
+
+# from pictures.models import PictureField
+
+
+class Startup(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    company_name = models.CharField("Company Name", max_length=64)
+
+    founders = models.ManyToManyField(Founder)
+
+    STARTUP_SIZE = (("S", "Small"), ("M", "Medium"), ("B", "Big"), ("L", "Large"))
+
+    size = models.CharField(
```

Comment:
Consider if ```blank=True``` is necessary, as it allows the field to be saved as an empty string in the database, which might be redundant with the ```default``` value

Reply:
Fixed in 00be11774de59a7e0960628bb8455c8bfa06548d